doc/release_notes/v2.1.0.rst

```diff
@@ -62,6 +63,32 @@ Changes
 * Invalid values used with the :class:`~pydicom.dataset.Dataset`
   :func:`in<operator.__contains__>` now raise :class:`ValueError` rather than
   returning ``False`` (:issue:`1200`)
+* Data dictionaries updated to version 2020c of the DICOM Standard
+* The following UID constants are deprecated and will be removed in v2.2:
+
+  * ``JPEGBaseline`` (use :attr:`~pydicom.uid.JPEGBaseline8Bit` instead)
+  * ``JPEGExtended`` (use :attr:`~pydicom.uid.JPEGExtended12Bit` instead)
+  * ``JPEGLossless`` (use :attr:`~pydicom.uid.JPEGLosslessSV1` instead)
+  * ``JPEGLSLossy`` (use :attr:`~pydicom.uid.JPEGLSNearLossless` instead)
+  * ``JPEG2000MultiComponentLossless`` (use
+    :attr:`~pydicom.uid.JPEG2000MCLossless` instead)
+  * ``JPEG2000MultiComponent`` (use :attr:`~pydicom.uid.JPEG2000MC` instead)
+
+* Starting in v2.2 the value for ``JPEGLossless`` will change from
+  1.2.840.10008.1.2.4.70 to 1.2.840.10008.1.2.4.57 to match its UID keyword
+* The following lists of UIDs are deprecated and will be removed in v2.2:
+
```

Comment:
Technical point - we have removed before like this, but I'm mindful of the discussions around allowing breaking changes only on major version changes.  Removal could break old code, and in this case I think there is no harm in leaving in them in until v3.0.  The one `JPEGLossless` identifier can't be helped, but the others could.

Reply:
Done, left to v3.0